src/common/types.ts

```diff
@@ -564,20 +567,7 @@ interface GamepadActionArgsWithoutMetadata {
   metadata?: undefined
 }
 
-type ElWebview = {
-  canGoBack: () => boolean
-  canGoForward: () => boolean
-  goBack: () => void
-  goForward: () => void
-  reload: () => void
-  isLoading: () => boolean
-  getURL: () => string
-  copy: () => string
-  selectAll: () => void
-  findInPage: (text: string | RegExp) => void
-}
-
-export type WebviewType = HTMLWebViewElement & ElWebview
+export type WebviewType = HTMLWebViewElement & WebviewTag
```

Comment:
We can get rid of this line and just import `WebviewTag` directly wherever we need it. It inherits from `HTMLElement` the same as HTMLWebViewElement